### PR TITLE
Droth 144 OTH integration to Tierekisteri - stops

### PR DESCRIPTION
**Pull request by leirasf**
Implementation of the following tasks:
- DROTH - 243 roska-astia and koroke are added to the equipment information of stops;
- DROTH - 244 Generating LiVi identifier to stops in OTH;
- DROTH - 253 Notification than equipment information can disappear when mass transit stop is transferred from municipality to ELY;
- DROTH-252 Show pop-up if mass transit stop is moved more than 50 meters

### CODE REVIEW

**Comment by leirasf** [comment]
It looks good to me

**Comment by marquesfm** [comment]
Looks good for me.

File: UI/src/view/MassTransitStopForm.js [hunk 1]
```diff
@@ -521,6 +523,14 @@
         streetViewHandler.update();
       });
 
+      eventbus.on('assetPropertyValue:changed', function (event) {
+        var property = event.propertyData;
+
+        if(property.publicId == 'tietojen_yllapitaja' && (_.find(property.values, function (value) {return value.propertyValue == '2';}))){
```
Comment (halied) [inline UI/src/view/MassTransitStopForm.js]: Should there be === instead of == ?

Comment (marquesfm) [inline UI/src/view/MassTransitStopForm.js]: Hanna, modification done.

**Comment by leirasf** [comment]
New changes looks good

**Comment by marquesfm** [comment]
Looks ok for me

**Pull request merged**